source/_integrations/august.markdown

```diff
@@ -69,6 +72,10 @@ If you have an August Smart Lock with DoorSense, once you have enabled the Augus
 
 - Door sensor
 
+## Button
+
+A button is created to wake the lock from a deep sleep. If your lock is not reporting a status, it may be in a deep sleep, and the button can be used to wake it. Locks are not automatically woken from deep sleep to preserve battery life.
```

Comment:
```suggestion
Buttons are created to wake locks from a deep sleep. If your lock is not reporting a status, it may be in a deep sleep, and the button can be used to wake it. Locks are not automatically woken from deep sleep to preserve battery life.
```